TMP2022/technical-writing-track/joy-tw-owans/README.md

```diff
@@ -0,0 +1,140 @@
+
+The bank is designed especially for smartphone users. You can open an account within few minutes, track your spending, save more, and make the right money moves. The idea of Kuda is to ease Nigerians from the frustrations they experience with the regular banks, hence the slogan - “the bank of the free”. You get a debit card for free, send money with no charges, and receive yearly interest when you save automatically.
+
+## Prerequisites <a name="prerequisites"></a>
+These are the conditions you need to meet to open a Kuda Account:
+
+* Be a minimum of 16 years old and a Nigerian citizen
+* A smartphone that supports the latest version of the KUDA app (for this guide, we’re using Android)
+* An active phone number and email address
+
+## How to signup <a name="howtosignup"></a>
+
+To sign up quickly and create an account on the Kuda Bank app, follow these steps below:
+
+#### 1. Download the app on Google Play Store and launch it. <a name="downloadtheappongoogleplaystoreandlaunchit"></a>
+#### 2.  Open the app and follow the prompts on your screen to open an account. <a name="opentheappandfollowthepromptsonyourscreentoopenanaccount"></a>
+    2.1   Click Open An Account
+    2.2   Enter your Email Address, Password, and Confirm password (Re-type the password)
+        
+    N/B: Ensure to use a strong password comprising of alphanumeric characters and symbols
+ 
+    2.3   If you were referred by someone and have a referral code, you can type the code. Else, skip that and click Next
+    2.4   Now, you’ll have to confirm the email address you imputed previously. Get the Verification Code from your email and enter it. Click Next
+    
+    N/B: If, for some reason, you checked your email and didn’t see the code, you can select Resend code.
+
+![](https://i.imgur.com/Nn0p86m.png)
+
+**Congratulations! You have successfully signed up and opened an account.**
+    
+#### 3. Setting up your profile <a name="settingupyourprofile"></a>
+    
+    3.1 Ensure to add your name correctly as it appears on your official documents and IDs. Click Next when done.
+        -    Legal First Name
+        -    Legal Middle Name (This is optional - only if you have a middle name)
+        -    Legal Last Name
+        -    Gender (Male or Female)
+        -    Date of Birth
+    
+    3.2 Enter your phone number correctly (Nigerian number) and click Next
+    3.3 Confirm your phone number by inputting the verification code sent to your SMS (just like you did with the email confirmation in step 2.4).
+    3.4 Fill in your house address correctly
+        -   State
+        -   LGA
+        -   City
+        -   House number
+        -   Street name
+        
+At this point, you have successfully created your account! You should see a “**Welcome to Kuda message**” on your screen. Click Continue!
+
+![](https://i.imgur.com/DogEbzw.png)
+
+#### 4. Sign in to your account <a name="signintoyouraccount"></a>
```

Comment:
Show yourself actively doing these so the user doesn't feel bored or along when going through this guide